docs/UserGuide.md

```diff
@@ -178,21 +184,21 @@ Some of TA Toolkit's commands require parameters, which have specific formats. B
+| INDEX         | -      | Positive integers only                                                                                                                                                                                                                                                                         | `1`, `2`, `3`                                   |
+| KEYWORD       | -      | Alphanumeric only                                                                                                                                                                                                                                                                              | `Alex`, `Bernice`                               |
+| NAME          | n/     | Alphanumeric, can contain multiple words                                                                                                                                                                                                                                                       | `Alex Yeoh`, `Bernice Yu`                       |
+| PHONE_NUMBER  | p/     | Numeric, minimum length of 3 digits                                                                                                                                                                                                                                                            | `99999999`, `88888888`                          |
+| TELEGRAM      | t/     | Starts with @, followed by alphanumeric characters or underscores                                                                                                                                                                                                                              | `@nus_cs2103` , `@johnDoe`                      |
+| EMAIL         | e/     | Includes letters, numbers, underscores, dots, and dashes, followed by the '@' symbol, then the domain name which includes at least one dot, separating the domain and the top-level domain (e.g., '.com', '.net'). The domain and the top-level domain must both have at least two characters. | `damithch@comp.nus.edu.sg`, `dcsdcr@nus.edu.sg` |
+| GITHUB        | g/     | Consists of alphanumeric characters, underscores, periods, and hyphens                                                                                                                                                                                                                         | `johnDoe`, `alice-yu`                           |
+| CLASS_GROUP   | c/     | Consists of alphanumeric characters, separated by hyphens                                                                                                                                                                                                                                      | `CS2100-T10`, `F14-3-2`                         |
```

Comment:
Could you change `F14-3-2` to `T42`, since some users might get confused about the Class_Group. We should make the class groups consistent with CS2100 usage.

Reply:
Also, could u change all references to CS2103T class groups to CS2100. Like F14-3 to T42.

Reply:
Also, for the delete note section, could you add this line.

+ If multiple indices are provided, they should be comma-separated.